runtime/compiler/runtime/JITServerAOTDeserializer.hpp

```diff
@@ -153,13 +152,14 @@ class JITServerAOTDeserializer
    PersistentUnorderedMap<J9Class *, uintptr_t/*ID*/> _classPtrMap;
    TR::Monitor *const _classMonitor;
 
-   PersistentUnorderedMap<uintptr_t/*ID*/, uintptr_t/*SCC offset*/> _methodMap;
+   PersistentUnorderedMap<uintptr_t/*ID*/, J9Method *> _methodIdMap;
+   PersistentUnorderedMap<J9Method *, uintptr_t> _methodPtrMap;
    TR::Monitor *const _methodMonitor;
 
-   PersistentUnorderedMap<uintptr_t/*ID*/, uintptr_t/*SCC offset*/> _classChainMap;
+   PersistentUnorderedMap<uintptr_t/*ID*/, uintptr_t * /*deserializer chain*/> _classChainMap;
```

Comment:
These chains (and the ones in the well-known classes map) could just be vectors, I suppose.